intro-mini-example/index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+  <head>
+  	<script src="https://d3js.org/d3.v4.min.js" charset="utf-8"></script>
+  	<link rel="stylesheet" href="main.css">
+  </head>
+  
+  <body>
+    <script>
+
```

Comment:
Maybe some comments to explain what you mentioned today?

Reply:
alrighty - added!